### ISSUE TITLE
Find out what the best metric is to evaluate the clusterings that we create

### ISSUE THREAD

**Issue by benvanwerkhoven** [nlesc-sherlock/cluster-analysis#7]
At the end of camera_identification.py we currently print a couple of metrics to evaluate the flat clustering produced by the program. We need to investigate which metrics are actually best to evaluate the clusterings produced by our application. Currently, we print the adjusted rand score, adjusted mutual info score, and the homogeneity, completeness, and v-measure scores using the metrics module of sklearn. I also think that, given the application, a low false positive rate is also very important.
